artemis-server/src/main/java/org/apache/activemq/artemis/core/server/embedded/EmbeddedActiveMQ.java

```diff
@@ -144,9 +155,11 @@ protected void initStart() throws Exception {
          activeMQServer = new ActiveMQServerImpl(configuration, mbeanServer, securityManager);
       }
 
-      URL brokerPropertiesFromClasspath = this.getClass().getClassLoader().getResource(ActiveMQDefaultConfiguration.BROKER_PROPERTIES_SYSTEM_PROPERTY_NAME);
-      if (brokerPropertiesFromClasspath != null) {
-         activeMQServer.setProperties(new File(brokerPropertiesFromClasspath.toURI()).getAbsolutePath());
+      if (propertiesResourcePath != null) {
+         URL brokerPropertiesFromClasspath = this.getClass().getClassLoader().getResource(propertiesResourcePath);
+         if (brokerPropertiesFromClasspath != null) {
+            activeMQServer.setProperties(new File(brokerPropertiesFromClasspath.toURI()).getAbsolutePath());
+         }
```

Comment:
Feels like this should really be in a separate commit against ARTEMIS-3627 specifically, where the original logic was only just added.

Reply:
It will get referenced via the commit message

Reply:
It should be the primary target of the commit, and shouldnt have unrelated test changes for another issue.

Reply:
pushed https://github.com/apache/activemq-artemis/pull/3924 with this logic and sorts the test failure. thanks!